libres/tests/analysis/test_update.cpp

```diff
@@ -0,0 +1,146 @@
+        rng_type *rng = rng_alloc(MZRAN, INIT_DEFAULT);
+
+        model true_model{a_true, b_true};
+
+        for (int iens = 0; iens < ens_size; iens++)
+            ens.push_back(model(rng));
+
+        const char *obs_key = "OBS1";
+        meas_block_type *mb =
+            meas_data_add_block(meas_data, obs_key, 1, obs_size);
+        obs_block_type *ob =
+            obs_data_add_block(obs_data, obs_key, obs_size, nullptr, false);
+
+        for (int iobs = 0; iobs < obs_size; iobs++) {
+            double x = iobs;
+            for (int iens = 0; iens < ens_size; iens++) {
+                const auto &m = ens[iens];
+                meas_block_iset(mb, iens, iobs, m.eval(x));
+            }
+
+            obs_block_iset(ob, iobs, true_model.eval(x), 0.25);
+        }
+
+        matrix_type *E = obs_data_allocE(obs_data, rng, ens_size);
+
+        int nparam = 2;
+        matrix_type *A = matrix_alloc(nparam, ens_size);
+        for (int iens = 0; iens < ens_size; iens++) {
+            const auto &model = ens[iens];
+            matrix_iset(A, 0, iens, model.a);
+            matrix_iset(A, 1, iens, model.b);
+        }
+
+        std::unordered_map<std::string, matrix_type *> parameters = {{"A", A}};
+
+        WHEN("Running update without rowscaling") {
+            analysis::run_analysis_update_without_rowscaling(
+                ies_module, ens_mask, meas_data, obs_data, rng, E, parameters);
+
+            // The goal here is to show that the accuracy of estimation inscreases
+            // with increasing ensemble size.
+            THEN("The average of the parameter distribution should be close to "
+                 "the true parameter value") {
+                if (ens_size == 10) {
+                    // REQUIRE(std::abs(a_true - matrix_get_row_sum(A, 0) /
+                    //                               ens_size) < 0.9);
+                    // REQUIRE(std::abs(b_true - matrix_get_row_sum(A, 1) /
+                    //                               ens_size) < 0.1);
+                    std::cout << "ens_size: " << ens_size << std::endl;
+                    std::cout << "a_diff: " << std::abs(a_true -
+                                          matrix_get_row_sum(A, 0) / ens_size)
+                              << std::endl;
+                    std::cout << "b_diff: " << std::abs(b_true -
+                                          matrix_get_row_sum(A, 1) / ens_size)
+                              << std::endl;
+                    std::cout << std::endl;
+                } else if (ens_size == 100) {
+                    // REQUIRE(std::abs(a_true - matrix_get_row_sum(A, 0) /
+                    //                               ens_size) < 0.12);
+                    // REQUIRE(std::abs(b_true - matrix_get_row_sum(A, 1) /
+                    //                               ens_size) < 0.015);
+                    std::cout << "ens_size: " << ens_size << std::endl;
```

Comment:
When the `fmt` library is included as a dep I would recommend using 
```C++
fmt::print("ens_size: {}\n", ens_size);
```
instead of `std::cout <<`

Reply:
I might as well get used to using the `fmt` library, even for simple testing like this. Thanks.